build.gradle

```diff
@@ -34,6 +34,7 @@ repositories {
     mavenLocal()
     // Remove the commented code below once TransportService is published to maven
     //maven { url "https://aws.oss.sonatype.org/content/repositories/snapshots" }
+    maven { url "https://d1nvenhzbhpy0q.cloudfront.net/snapshots/lucene/"}
```

Comment:
Need to add lucene dependencies in the build.gradle dependencies

Reply:
OpenSearch-3.0 SNAPSHOT has some dependencies on lucene. 
This is a work around to get our builds working. But we should figure out a long term solution.